crazyclock.ino

```diff
@@ -61,17 +75,18 @@ void setup() {
 
   beginLCD(&lcd, LCD_COLS, LCD_ROWS);
   beginRTC(&lcd, &rtc);
-  bool wifiAvailable = isWiFiAvailable(&lcd, ssid, password);
+  // bool wifiAvailable = isWiFiAvailable(&lcd, ssid, password);
+  bool wifiAvailable = false;
   if (wifiAvailable) {
 
-    isNtpAvailable = retrieveEpochTimeFromNTP(lcd, &epochSeconds);
+    bool isNtpAvailable = retrieveEpochTimeFromNTP(lcd, &epochSeconds);
     if (isNtpAvailable) {
 
       Serial.println("Updating RTC with time from NTP...");
       LocalDateTime localDateTime = plDateTimeConverter.fromUtc(epochSeconds);
-      mH = localDateTime.getLocalTimeFragment(HOURS);
-      mM = localDateTime.getLocalTimeFragment(MINUTES);
-      mS = localDateTime.getLocalTimeFragment(SECONDS);
+      int mH = localDateTime.getLocalTimeFragment(HOURS);
```

Comment:
I can see, you moved the definitions, but IMHO this won't work as they won't be visible in` loop {}`

Reply:
I moved it here, because those variables are not needed in `loop()` function right now and they might not be needed there. If they will, then I will revert the change. But in the current state they can be here.